run-videochat.sh

```diff
@@ -138,7 +138,7 @@ iw_server_is_started() {
 }
 
 module_id_by_sinkname() {
-    pacmd list-sinks | grep -e 'name:' -e 'module:' | grep -A1 "name: <$1>" | grep module: | cut -f2 -d: | tr -d ' '
+    pactl list sinks | grep -e "Name:" -e "Module:" |grep -A1 "Name: $1" | grep Module: | cut -f2 -d: | tr -d ' '
```

Comment:
Would this break the script for existing PulseAudio users? If so, there needs to be some way to switch between PulseAudio and pipewire.

Reply:
testing in a clean ubuntu install with only pulseaudio installed, both commands give the same output. so it will be backwards compatible for pulseaudio users
`command 'pactl' from deb pulseaudio-utils (1:13.99.2-1ubuntu2.1)`

Reply:
I tested your version in my Ubuntu system and it seems to be fine, indeed. Sorry, I just wasn't aware of `pactl` being part of the PA distribution as well :-).